packages/react-scripts/template-hf/index.js

```diff
@@ -1,16 +1,20 @@
 // This file is meant to be run by Heroku. It can be run locally
-// by typing heroku local on the command line.
+// by typing `heroku local` on the command line.
+
+// You should not need to modify this file.
+// Changes to proxies should be done in setupProxy.js
+// Changes to routes should be done in server.js
 
 // sets local proxies using http-proxy-middleware configs
-// docs here: https://github.com/fs-webdev/exo#proxy
+// docs here:
```

Comment:
were you going to add anything after `docs here:` ? or was that intentional?

Reply:
Meant to delete the whole line.